SpywareFilter/sections/general_url.txt

```diff
@@ -5,6 +5,11 @@
 ! Bad: .org/ads/$domain=example.org (for instance, should be in speficic.txt)
 !
 !
+! server-side GTM pattern B ex. https://load.ss.esade.edu/ykpgwaiz.js?st=5V4W3ZR
+!+ NOT_PLATFORM(ios, ext_safari)
+/\/\/([0-9a-z]+\.){1,}[0-9a-z]+\.[a-z]{2,}\/[a-z]{8}\.js\?st=[0-9A-Z]{6,8}$/$script,~third-party,match-case
+!+ PLATFORM(ios, ext_safari)
+/\/\/([0-9a-z]+\.)?[0-9a-z]+\.[0-9a-z]+\.[a-z][a-z]+\/[a-z]{8}\.js\?st=[0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z]$/$script,~third-party,match-case
```

Comment:
`{8}` missed.
Cannot find a site with such tracker, but in the main safari tab this rule works only without `script` modifier (`document` in safari dev tools)
`/\/\/([0-9a-z]+\.)?[0-9a-z]+\.[0-9a-z]+\.[a-z][a-z]+\/[a-z][a-z][a-z][a-z][a-z][a-z][a-z][a-z]\.js\?st=[0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z]$/$~third-party,match-case`
replaced `{8}` by 8x `[0-9A-Z]`

Reply:
> `{8} missed.`

Intentional, if we want to cover all we have to use `st=[0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z]+` which matches to even `100`. Maybe fine, IDK. Not sure which number is the most popular, but you can find with `https://publicwww.com/websites/%22.js%3Fst%3D%22/`.

Reply:
Updated

Reply:
Oops, `st=[0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z][0-9A-Z]?[0-9A-Z]?$` should work, right? Didn't notice such an easy trick.